2.7.0/todo/result_onnx.md

```diff
@@ -28,74 +28,101 @@ The categories below are as follows:
 
 ### bc breaking
 
-- [ONNX] Set USE_EXPERIMENTAL_LOGIC to True ([#137296](https://github.com/pytorch/pytorch/pull/137296))
+#### `torch.onnx.dynamo_export` now uses the ExportedProgram logic path (#137296)
+
+Users using the `torch.onnx.dynamo_export` API may see some `ExportOptions` become
```

Comment:
are you able to be more specific about what export options? And can you include the error the user is expected to get in 2.7 when using one of the not supported options anymore? This way, someone can quickly google search the bad result!